src/CMakeLists.txt

```diff
@@ -3,11 +3,13 @@
 
 find_package(PkgConfig REQUIRED)
 
-pkg_check_modules(Glew REQUIRED IMPORTED_TARGET glew)
+if (RENDER) 
+	pkg_check_modules(Glew REQUIRED IMPORTED_TARGET glew)
```

Comment:
It looks like these are tabs, it would be great if we could stick to spaces all through the files if possible (most of our code sticks to two spaces indent for CMake and C++, like LLVM).